src/schema/type/DefinedTerm.graphql

```diff
@@ -0,0 +1,70 @@
+"https://pending.schema.org/DefinedTerm"
+type DefinedTerm implements ThingInterface {
+    ### Metadata properties ###
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://purl.org/dc/elements/1.1/contributor,https://schema.org/contributor"
+    contributor: String
+    "http://purl.org/dc/elements/1.1/coverage"
+    coverage: String
+    "http://purl.org/dc/elements/1.1/creator"
+    creator: String!
+    "http://purl.org/dc/elements/1.1/date"
+    date: _Neo4jDate
+    "http://purl.org/dc/elements/1.1/description,https://schema.org/description"
+    description: String
+    "https://schema.org/disambiguatingDescription"
+    disambiguatingDescription: String
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    "http://purl.org/dc/elements/1.1/publisher"
+    publisher: String
+    "http://purl.org/dc/elements/1.1/relation"
+    relation: String
+    "http://purl.org/dc/elements/1.1/rights"
+    rights: String
+    "http://purl.org/dc/elements/1.1/source"
+    source: String
+    "http://purl.org/dc/elements/1.1/subject"
+    subject: String
+    "http://purl.org/dc/elements/1.1/title"
+    title: String
+    "http://purl.org/dc/elements/1.1/type,https://www.w3.org/2000/01/rdf-schema#type"
+    type: String
+    "http://purl.org/dc/elements/1.1/date"
+    created: String
+    "http://purl.org/dc/elements/1.1/date"
+    updated: String
+    #################################
+    ### ThingInterface properties ###
+    "https://schema.org/additionalType"
+    additionalType: [String]
+    "https://schema.org/additionalProperty"
+    additionalProperty: [PropertyValue] @relation(name: "ADDITIONAL_PROPERTY", direction: "OUT")
+    "https://schema.org/alternateName"
+    alternateName: String
+    "https://schema.org/image"
+    image: URL
+    "https://schema.org/mainEntityOfPage"
+    mainEntityOfPage: [CreativeWorkInterface] @relation(name: "MAIN_ENTITY_OF_PAGE", direction: "OUT")
+    "https://schema.org/name"
+    name: String
+    "https://schema.org/potentialAction"
+    potentialAction: [ActionInterface] @relation(name: "POTENTIAL_ACTION", direction: "OUT")
+    "https://schema.org/sameAs"
+    sameAs: URL
+    "https://schema.org/subjectOf"
+    subjectOf: [CreativeWorkInterface] @relation(name: "SUBJECT_OF", direction: "OUT")
+    "https://schema.org/url"
+    url: URL
+    ##############################
+    ### DefinedTerm properties ###
+    "https://pending.schema.org/termCode"
+    termCode: String!
+    # TODO: inDefinedTermSet can also be a URL, is this needed in the CE?
+    # TODO: This should only be a relation to one DefinedTermSet
+    "https://pending.schema.org/inDefinedTermSet"
+    inDefinedTermSet: [DefinedTermSet] @relation(name: "HAS_DEFINED_TERM", direction: "IN")
```

Comment:
In a database, I'd model this as a one-to-many relation. A term can belong to only one term set. A term set can contain many terms. I don't know how to model this with the neo4j-graphql library, and so it's currently a many-to-many, though this is wrong. Any suggestions on how to make this better would be appreciated

Reply:
Yes, I've also encountered this problem for the `EntryPoint#potentialAction` property. The same `ControlAction` can be linked multiple times to the same EntryPoint. Unfortunately, there is no easy solution with the `neo4j-graphql-js` library. 

[Neo4j constraints](https://neo4j.com/docs/cypher-manual/current/administration/constraints/) could be the answer to this problem.

Reply:
In the case of a neo4j constraint we would have to see what the error-case is if you try and add more than one item here. Will it be a regular graphql data error, or will the server crash :)

#92

Reply:
That is a good question indeed 😄